src/commands/_blocks.live.ts

```
@@ -93,11 +93,28 @@ export async function startLiveMonitoring(config: LiveMonitoringConfig): Promise
 
 		// Handle and display errors
 		const errorMessage = error instanceof Error ? error.message : String(error);
-		terminal.startBuffering();
-		terminal.clearScreen();
-		terminal.write(pc.red(`Error: ${errorMessage}\n`));
-		terminal.flush();
-		logger.error(`Live monitoring error: ${errorMessage}`);
+		
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)